packages/experimental/src/components/AboutModal/AboutModal.js

```diff
@@ -69,52 +68,47 @@ export const AboutModal = ({
         </div>
       </ModalBody>
       <ModalFooter>
-        {technologiesUsed && technologiesUsed.length ? (
-          <Tabs className={`${pkg.prefix}-about-modal-tab-container`}>
-            <Tab
-              id="about-modal-technologies-used-tab"
-              label="Technologies used">
-              <div className={`${pkg.prefix}-about-modal-tab-content-flex`}>
-                {technologiesUsed &&
-                  technologiesUsed.length &&
-                  technologiesUsed.map((tech) => (
-                    <img
-                      key={tech.alt}
-                      src={tech.src}
-                      alt={tech.alt}
-                      className={`${pkg.prefix}-about-modal-tech-used-item`}
-                    />
-                  ))}
-              </div>
-            </Tab>
-            <Tab id="about-modal-version-number-tab" label="Version number">
-              <div
-                className={`${pkg.prefix}-about-modal-tab-content-flex ${pkg.prefix}-about-modal-tab-content-version-flex`}>
-                <p className={`${pkg.prefix}-about-modal-version-label`}>
-                  Version number
-                </p>
-                <p className={`${pkg.prefix}-about-modal-version-number`}>
-                  {versionNumber}
-                </p>
-              </div>
-            </Tab>
-          </Tabs>
-        ) : (
-          <>
-            <p className={`${pkg.prefix}-about-modal-version-label`}>
-              Version number
-            </p>
-            <p className={`${pkg.prefix}-about-modal-version-number`}>
-              {versionNumber}
-            </p>
-          </>
-        )}
+        {additionalInfo &&
```

Comment:
```suggestion
        {additionalInfo && additionalInfo.length > 0
```

Just to protect against some numpty doing additionalInfo=[]. Empty arrays aren't falsy.

Reply:
Done.